ci-operator/step-registry/mtr/execute-ui-tests/mtr-execute-ui-tests-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Set the TARGET_URL value using the cluster_url
+TARGET_URL="http://mtr-mtr.$(cat ${SHARED_DIR}/cluster_url)"
+
+# Installing Cypress
+# This needs to happen in this script rather than the Dockerfile because
+# of some issues with permissions in the container caused by OpenShift
+echo "Installing Cypress"
```

Comment:
Why isn't this done in the dockerfile?

Reply:
If we install it in the Dockerfile, we get permission errors. Cypress is installed by root in the Dockerfile, but OpenShift pods don't run as root, so it doesn't recognize the Cypress install from the Dockerfile

Reply:
Ok thanks for the info, the ACM team is also running Cypress, installed from the [Dockerfile](https://github.com/stolostron/clc-ui-e2e/blob/main/Dockerfile.interops) so I was curious, though the install is a bit different.

They use a [package.json](https://github.com/stolostron/clc-ui-e2e/blob/main/package.json#L48) file to install cypress through devDependencies it looks like. 

Something is allowing cypress to run without needing to be root using this method but I'm not positive what it is yet

Reply:
It looks like MTR PQE has it in [their `package.json` file](https://github.com/windup/windup-ui-tests/blob/main/package.json), but it's in `dependencies`, not it `devDependencies`. I wonder if `devDependencies` packages install will more permissions for developers to play with things.

I would be happy to remove it, but I don't feel comfortable changing the `package.json`. If @sshveta or @stillalearner want to make that change, we can give it a shot in our scenario.

That being said, I don't think it will cause any issues. More of a nice to have.

Reply:
So I spoke too soon when I commented on this. When I tried the Cypress install from the dockerfile it lead to a very root based install and then their test failed to access necessary files in the env.

I submitted this PR to their Dockerfile and it resolved the issues that I was seeing.

We could potentially extend the Dockerfile image with this type of solution then remove this from the commands.sh. The biggest benefit that we get by doing this is that this execute test pod will never fail due to a cypress installation issue which is possible right now.

I'd agree that this would be classified as a low priority/nice to have task.

Reply:
@ascerra Can you link me to your PR to their Dockerfile? I can make the change, we have some time before we can merge this anyway

Reply:
@ascerra This should be resolved [[LPTOCPCI-100]](https://issues.redhat.com/browse/LPTOCPCI-100)